src/wizard/deploy.jl

```diff
@@ -1,7 +1,12 @@
 function print_build_tarballs(io::IO, state::WizardState)
-    urlhashes = zip(state.source_urls, state.source_hashes)
-    sources_string = strip(join(map(urlhashes) do x
-        string(repr(x[1])," =>\n    ", repr(x[2]), ",\n")
+    urlfiles = zip(state.source_urls, state.source_files)
+    sources_string = strip(join(map(urlfiles) do x
+        url_string = repr(x[1])
+        if endswith(url_string, ".git")
+            "GitSource($(url_string), $(repr(x[2].hash)))"
+        else
+            "FileSource($(url_string), $(repr(x[2].hash)))"
+        end
```

Comment:
Yeah, this is smarter and cleaner to do this kind of splitting in the wizard, rather than at `build_tarballs()` time.